webpack.config.js

```diff
@@ -0,0 +1,49 @@
+// const path    = require("path")
+// const webpack = require("webpack")
+
+// module.exports = {
+//   mode: "production",
+//   devtool: "source-map",
+//   entry: {
+//     application: "./app/javascript/application.js"
+//   },
+//   output: {
+//     filename: "[name].js",
+//     sourceMapFilename: "[file].map",
+//     path: path.resolve(__dirname, "app/assets/builds"),
+//   },
+//   plugins: [
+//     new webpack.optimize.LimitChunkCountPlugin({
+//       maxChunks: 1
+//     })
+//   ]
+// }
```

Comment:
- [ ] kindly consider removing all commented codes to  make your code base look more clean and professional